tox.ini

```diff
@@ -226,7 +226,7 @@ commands_pre =
   test: pip install {toxinidir}/opentelemetry-python-core/opentelemetry-sdk[test]
   test: pip install {toxinidir}/opentelemetry-python-core/tests/util[test]
 
-  test: pip install {toxinidir}/opentelemetry-instrumentation[test]
+  test: pip install yarl {toxinidir}/opentelemetry-instrumentation[test]
```

Comment:
Do you know why you added yarl here, specifically? It seems like it looks different from the rest of the lines near it, so just wondering.

Reply:
I'd also suggest installing that package on its own line.

Reply:
I agree. These lines seem to be there specifically for installing OTel packages. Adding external dependencies should be done through either `requirements.txt` (if it's a dev/prod dependency) or through the `deps` field for that particular test in `tox.ini` (if it's only required for the test).

Reply:
Sounds good. I'm still trying to find the right place where the imports are supposed to go so all the tests and docker tests succeed but I'll dig into it a little more.